### PR TITLE
Added support to use imagePullSecrets defined in values.yaml file

### PR DESCRIPTION
**Pull request by navneet066**
in fluentbit-operator deployment file, there was no support for the imagePullSecrets though it was available in values.yaml file. I will add in other templates also

### CODE REVIEW

**Comment by benjaminhuo** [comment]
@navneet066 Thanks for the PR. Would you please squash your commits into one?

File: chart/fluentbit-operator/templates/fluentbit-operator-deployment.yaml [hunk 1]
```diff
@@ -45,3 +45,7 @@ spec:
         - name: env
           mountPath: /fluentbit-operator
       serviceAccountName: fluentbit-operator
+{{- if .Values.imagePullSecrets }}
```
Comment (wanjunlei) [inline chart/fluentbit-operator/templates/fluentbit-operator-deployment.yaml:48]: This format will be less readable, the following format will be better.

```yaml
      serviceAccountName: fluentbit-operator
      {{- if .Values.imagePullSecrets }}
      imagePullSecrets:
      {{ toYaml .Values.imagePullSecrets | indent 8 }}
      {{- end }}
```

**Review by wanjunlei — APPROVED** [review]

**Review by wenchajun — APPROVED** [review]

**Comment by benjaminhuo** [comment]
@navneet066 Would you please squash your 4 commits into one using cmd like `git rebase -i squash head~5`

**Comment by navneet066** [comment]
Hi Everyone, Something wrong with my git. I am closing this one and will create the same with the new PR .

**Pull request closed without merging**